pages/about.vue

```diff
@@ -50,3 +50,16 @@ export default Vue.extend({
   },
 });
 </script>
+<style scoped lang="scss">
+.content-row {
```

Comment:
maybe we can share this across components, or even put these margins on the `v-container` so we don't have to apply to each row. thoughts?